addons/l10n_zm/data/template/account.tax-zm.csv

```diff
@@ -0,0 +1,25 @@
+"id","description","invoice_label","type_tax_use","name","amount_type","amount","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id"
+"zm_tax_sale_export_0","Export Rate","0%","sale","0% (EX)","percent","0.0","tax_group_0","base","invoice",
```

Comment:
```suggestion
"zm_tax_sale_export_0","Export Rate","0%","sale","0% EX","percent","0.0","tax_group_0","base","invoice",
```
Probably parenthesis are not very useful, it will be more consistent with the tax taxonomy of other localizations.